src/EventDispatcher/SimpleEventDispatcher.php

```diff
@@ -12,7 +12,7 @@
 
 final class SimpleEventDispatcher implements EventDispatcherInterface
 {
-    /** @var array<int, Closure> */
+    /** @var Closure[] */
```

Comment:
`Closure[]` mean any array key type (string/int/resource)

Reply:
> `Closure[]` means any array key type (string/int)

Yes, can be int or string.
If use named arguments in constructor, then in array will be string keys.